Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml

```diff
@@ -25,3 +26,15 @@
     - state: green
     - sprite: DeltaV/Markers/jobs.rsi
       state: courier
+
+- type: entity
```

Comment:
this should be on noSpawn: true

Reply:
Why so?

Reply:
> Why so?

my brain rotted temporarily and forgot this is a job marker and not an antag marker